### PR TITLE
Update schema.md - reference introduction

### PR DESCRIPTION
**Pull request by stufraser1**
Update introduction section to better reflect key elements of each component. Unsure whether class diagram should remain or be replaced with something derived directly from updated schema

**Related issues**

 addresses https://github.com/GFDRR/rdl-standard/issues/148

**Merge checklist**

- [x] Update the changelog ([style guide](developer_docs.md#changelog-style-guide))
- [x] Run `./manage.py` pre-commit

### CODE REVIEW

**Review by matamadio — APPROVED** [review]

File: docs/reference/schema.md [hunk 1]
```diff
@@ -10,14 +10,14 @@ This page presents the schema in tables with additional information in paragraph
...
+- [Hazard](#hazard): including the main hazard type and process, triggering hazard and process, hazard intensity units, occurrence frequency of individual events, multiple hazard footprints per event, historical and stochastic events sets, and analytical methods used.
+- [Exposure](#exposure): including asset category (including human, built environment assets and natural assets), taxonomy used to describe characteristics of assets and people, cost type, location, geometry types.
+- [Vulnerability](#vulnerability): describing vulnerability and fragility relationships and indexes, which estimate the impact on people or assets as a function of hazard intensity. This component uses attributes consistent with the hazard, exposure and loss components.
+- [Loss](#loss): modelled monetary and non-monetary damage and losses produced in a risk assessment, including the ability to link losses to the hazard, exposure, and vulnerability components used in the analysis. Allows description of common impact and risk metrics for direct and indirect impacts, and for individual historical and hypothetical events, and for large events sets in year loss tables and event loss tables.
```
Comment (odscjen) [inline docs/reference/schema.md:18]: You've described these 4 components in 2 different ways. Hazard and Exposure are essentially just listing the fields and objects within the components. Vulnerability and Loss are more descriptive as to what these components represent and what the actual data is which could be read as implying that the schema covers the data. It doesn't, it covers the metadata that describes the data.

I'd suggest picking one type of description and using that for all 4. Given that the Glossary contains (or will contain) definitions for all 4 component concepts I'd recommend using this section as a description of what the components contain, i.e. the details of what data can be described using each component.

Comment (stufraser1) [reply docs/reference/schema.md]: Resolved in ad586d2 commit. Also made other descriptions shorter and more targeted.

File: docs/reference/schema.md [hunk 2]
```diff
@@ -2,22 +2,23 @@
...
 
-- [Hazard](#hazard): main hazard type, specific process, trigger of the hazard, occurrence frequency of event, intensity unit to measure the process and analytical method.
-- [Exposure](#exposure): asset category, specific taxonomy, cost type and value.
-- [Vulnerability](#vulnerability): model that links hazard intensity and exposure classification to measure of impact over the total exposed value.
-- [Loss](#loss): modelled damage and losses produced in a risk assessment as a function of hazard, exposure and vulnerability components.
+- [Hazard](#hazard): metadata to describe hazard data, including the main hazard type and process, triggering hazard and p rocess, hazard intensity units, occurrence frequency of individual events, multiple hazard footprints per event, historical and stochastic events sets, and analytical methods used.
+- [Exposure](#exposure): metadata describing exposure data, including asset category (including human, built environment assets and natural assets), taxonomy used to describe characteristics of assets and people, cost type, location, geometry types.
```
Comment (odscjen) [inline docs/reference/schema.md:16]:
```suggestion
- [Exposure](#exposure): metadata describing exposure data, including asset category (including human, built environment assets and natural assets), taxonomy scheme used to describe characteristics of assets and people, cost type, location and geometry types.
```

File: docs/reference/schema.md [hunk 3]
```diff
@@ -2,22 +2,23 @@
...
+- [Vulnerability](#vulnerability): metadata to describe vulnerability and fragility relationships and indexes, including type of impact, type of exposure, hazard intensity measure and approach used to develop the relationship. This component uses attributes consistent with the hazard, exposure and loss components.
+- [Loss](#loss): metadata describing monetary and non-monetary damage and losses produced in a risk assessment. This includes fields to link losses to the hazard, exposure, and vulnerability components used in the analysis. The loss component enables description of common impact and risk metrics for direct and indirect impacts, for individual historical and hypothetical events, and for large events sets in year loss tables and event loss tables.
 
-For definitions of these terms, please see the [Glossary](../glossary.md).
+
+For general definitions of hazard, exposure, vulnerability and loss, please see the [Glossary](../glossary.md).
```
Comment (odscjen) [inline docs/reference/schema.md:21]: Just a comment that Loss is currently missing from the Glossary

File: docs/reference/schema.md [hunk 4]
```diff
@@ -208,7 +200,9 @@ Schema attributes for an earthquake hazard map related to an occurrence probabil
 
 ## Exposure
 
-The exposure schema covers a wide variety of data describing structural, infrastructural and environmental asset, population, and socio-economic descriptors, each with relevant attributes for assessing risk from multiple hazards. The schema was developed based on [GEM Taxonomy 2.0](https://wiki.openstreetmap.org/wiki/GED4ALL) to accommodate the most important spatial features commonly employed in risk analysis to identify and estimate exposed value.
+The Exposure metadata describes datasets containing information on the distribution and characteristics of built environment assets (buildings and infrastructure), natural assets and population, used in risk assessment. The Exposure component provides codelists to describe the type of assets and costs, and the taxonomy scheme that is used to describe construction and demographic information contained in the dataset.
```
Comment (odscjen) [inline docs/reference/schema.md:203]:
```suggestion
The exposure component describes metadata for datasets containing information on the distribution and characteristics of built environment assets (buildings and infrastructure) and natural assets and population, that are used in risk assessment. The exposure component provides codelists to describe the type of assets and costs, and the taxonomy scheme that is used to describe construction and demographic information contained in the dataset.
```

File: docs/reference/schema.md [hunk 5]
```diff
@@ -286,15 +275,10 @@ Two exposure datasets are shown together in the example: building footprints pol
 
 ## Vulnerability
 
-The vulnerability schema includes physical fragility and vulnerability relationships in relation to specific hazards or for multi-hazard (combination of individual hazards). A wide range of model types and parameters can describe vulnerability, for this reason there are many possible variables accounted by the Vulnerability schema. But only a part of them will be required to describe one specific model.
-The schema distinguishes key information describing the vulnerability model, including:
+The vulnerability component describes fragility, damage-to-loss and vulnerability relationships and indexes for physical damage and social vulnerability that are used in risk analysis. It contains key information including the type of function, intensity and impact metrics used, which asset types or population groups it applies to, how it was developed and for what locations.
```
Comment (odscjen) [inline docs/reference/schema.md:278]:
```suggestion
The vulnerability component describes metadata for datasets that detail fragility, damage-to-loss and vulnerability relationships and indexes of physical damage and social vulnerability that are used in risk analysis. It contains key information including the type of function, intensity and impact metrics used, which asset types or population groups it applies to, how it was developed and for what locations. 
```

**Review by odscjen — CHANGES_REQUESTED** [review]
Fixes a few typos and added some suggestions to improve the consistency of the description and ensure that in every case users understand that the standard is only detailing metadata.

File: docs/reference/schema.md (hunk 3 above)
Comment (stufraser1) [reply docs/reference/schema.md:20]: addressing in https://github.com/GFDRR/rdl-standard/issues/157

**Review by stufraser1 — COMMENTED** [review]
Jen, can you check your comments have all been committed (some are outdated and I wanted to commit but couldn't)

File: docs/reference/schema.md [hunk 6]
```diff
@@ -88,7 +85,9 @@ addtargets:
 
 ## Hazard
 
-The hazard schema stores data about the intensity and occurrence probability of physical hazard phenomena such as floods, earthquakes, wildfires or others. The specific hazard process can be defined and measured with a specific intensity unit. For example, earthquake hazard may be represented as ground shaking, liquefaction or ground displacement.
+The hazard component describes metadata about modeled natural hazards data including hazard intensity footprints of historical or hypothetical events, return period hazard maps, hazard or susceptibility index, and stochastic event sets. The metadata defines hazard type, physical process and intensity measures used in the dataset. Multiple hazards and processes (including cascading events) can be defined for each hazard, enabling users to describe dataset that contain, for example, earthquake ground shaking and liquefaction, and tsunami inundation triggered by the earthquake.
```
Comment (odscjen) [inline docs/reference/schema.md:88]:
```suggestion
The hazard component describes metadata about modeled natural hazards data, including hazard intensity footprints of historical or hypothetical events, return period hazard maps, hazard or susceptibility index, and stochastic event sets. The metadata defines the hazard type, physical process and intensity measures used in the dataset. Multiple hazards and processes (including cascading events) can be defined for each hazard, enabling users to describe dataset that contain, for example, earthquake ground shaking and liquefaction, and tsunami inundation triggered by the earthquake.
```

File: docs/reference/schema.md [hunk 7]
```diff
@@ -286,15 +270,9 @@ Two exposure datasets are shown together in the example: building footprints pol
 
 ## Vulnerability
 
-The vulnerability schema includes physical fragility and vulnerability relationships in relation to specific hazards or for multi-hazard (combination of individual hazards). A wide range of model types and parameters can describe vulnerability, for this reason there are many possible variables accounted by the Vulnerability schema. But only a part of them will be required to describe one specific model.
-The schema distinguishes key information describing the vulnerability model, including:
+The vulnerability component describes fragility, damage-to-loss and vulnerability relationships and indexes for physical damage and social vulnerability that are used in risk analysis. It contains key information including the type of function, intensity and impact metrics used, which asset types or population groups it applies to, how it was developed and for what locations.
```
Comment (odscjen) [inline docs/reference/schema.md:273]:
```suggestion
The vulnerability component describes metadata for datasets that detail fragility, damage-to-loss and vulnerability relationships and indexes for physical damage and social vulnerability that are used in risk analysis. It contains key information including the type of function, intensity and impact metrics used, which asset types or population groups it applies to, how it was developed and for what locations.
```

**Review by odscjen — CHANGES_REQUESTED** [review]
Just a couple of minor copy editing changes left.

Also don't forget to update the change log and run ./manage.py prior to the final commit

**Comment by stufraser1** [comment]
> update the change log and run ./manage.py prior to the final commit

Please run pre-commit, I have updated changelog.

**Review by odscjen — APPROVED** [review]

**Pull request merged**